src/components/utils/test/date_time_test.cc

```diff
@@ -108,11 +108,10 @@ TEST(DateTimeTest, CalculateTimeSpan) {
   // arrange
   const TimevalStruct time = date_time::DateTime::getCurrentTime();
 
-  const uint32_t sleep_time_mSec = 10;
+  const uint64_t sleep_time_mSec = 100;
```

Comment:
@orhan-mehmedov , on QT platform, sleep(10) , sometimes works inncorect. thread can sleep only 1 ms.  I read that sleep() with values < 15 , can work wrong.